addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml

```diff
@@ -82,6 +75,46 @@
 		</config-description>
 	</thing-type>
 
+	<channel-group-type id="displayInformationGroup">
+		<label>Display</label>
+		<channels>
+			<channel id="displayInformation" typeId="displayInformationChannel" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone1Controls">
```

Comment:
@Stratehm I only came across this now (and had missed it when reviewing the PR): Why did you actually create 3 group-types for the 3 zones? The idea of the groups is that you only define them once and then reuse the same type for the 3 zones. Would it be possible to refactor that binding or is there any reason against doing so?